ccloud/docs/includes/connect-worker-override-policy.rst

```diff
@@ -0,0 +1,6 @@
+Finally, configure the Connect worker to allow overrides, because |crep| needs to override the default behavior of the Connect worker's embedded producer.
```

Comment:
```suggestion
Finally, configure the |kconnect| worker to allow overrides, because |crep| needs to override the default behavior of the |kconnect| worker's embedded producer.
```